packages/react/src/Button/Button.features.stories.tsx

```diff
@@ -61,9 +61,7 @@ export const TrailingCounter = () => {
   )
 }
 
-export const TrailingCounterWithNoText = () => {
-  return <Button leadingVisual={CommentIcon} count={3} />
-}
+export const TrailingCounterWithNoText = () => <Button leadingVisual={CommentIcon} count={3} />
```

Comment:
Should we add an aria-label or a Tooltip to this example?

Reply:
Hmmm I'm torn on this. I'd prefer not to encourage tooltips, but it's common to use them for icon-only buttons.

Maybe add a second example in the same story?